src/reducers/proposals.js

```diff
@@ -130,6 +130,29 @@ function extractProposals(topics, network) {
     }));
     return acc.concat(proposals);
   }, []);
+  return Promise.all(
+    proposals.map(async p => {
+      if (!p.govVote) {
+        try {
+          const [eta, executed] = await Promise.all([
+            window.maker.service('spell').getEta(p.source),
```

Comment:
note that this guy now throws if `eta` isn't set. I think that's fine, right? the error will be caught by the catch block and the keys below will stay `undefined`